indico/htdocs/js/indico/modules/events/cloning.js

```diff
@@ -0,0 +1,129 @@
+/* This file is part of Indico.
+ * Copyright (C) 2002 - 2017 European Organization for Nuclear Research (CERN).
+ *
+ * Indico is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Indico is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Indico; if not, see <http://www.gnu.org/licenses/>.
+ */
+
+ /* global getFormParams:false */
+
+
+(function(global) {
+    'use strict';
+
+    global.setupCloneDialog = function setupCloneDialog() {
+        var $formContainer = $('#event-clone-form-container');
+        var $form = $('#event-clone-form');
+        var $eventCount = $('#event-count').hide();
+        var $eventList = $form.find('.event-list');
+        var $cloneErrors = $('#clone-errors').hide();
+        var step = $formContainer.data('step');
+
+        var clonerDependencies = $formContainer.data('clonerDependencies');
+
+
+        function errorToHTML(error) {
+            if (typeof error == 'string') {
+                return error;
+            } else {
+                return $('<div>').append(error.map(function(item) {
+                    var label = $('<strong>').append(item[0]);
+                    var items = $('<ul>').append(item[1].map(function(message) {
+                        return $('<li>').text(message);
+                    }));
+                    return $('<div>').append(label, items);
+                }));
+            }
+        }
+
+        function updateCount() {
+            var $cloneButton = $('.clone-action-button');
+
+            $.ajax({
+                url: $formContainer.data('previewUrl'),
+                method: 'POST',
+                data: getFormParams($form),
```

Comment:
In the end you are submitting the form's data with AJAX. `$(someform).ajaxSubmit()` does exactly that, instead of having to use our custom `data: getFormParams($form)`